scripts/build_test.py

```diff
@@ -27,11 +29,10 @@
 import tempfile
 import threading
 
-# pylint: disable=relative-import
-import build
 from core.tests import test_utils
+import python_utils
 
-# pylint: enable=relative-import
+from . import build
```

Comment:
Looks like this works here, which is great; can we do it more generally?

Reply:
Yes I removed the pylint pragma and removed the absolute imports wherever I can.

Reply:
But why does "from . import XXX" work here when it doesn't in other places?

Reply:
It was only not working in python_utils_test and that's just because I forgot to add the absolute_import from future. I added it and its working. Hence the above pylint pragma is removed completely from the codebase